ddtrace/internal/telemetry/metrics.py

```diff
@@ -149,6 +146,6 @@ def to_dict(self):
         data = {
             "metric": self.name,
             "points": self._points,
-            "tags": ["%s:%s" % (k, v) for k, v in self._tags.items()],
+            "tags": self._tags,
```

Comment:
"tags": ["{}:{}".format(k, v).lower() for k, v in self._tags] if self._tags else [],